### ISSUE TITLE
Tabs for multiple servers

### ISSUE THREAD

**Issue by TheNightmanCodeth** [TheNightmanCodeth/mission#7]
The user should be able to open multiple servers in different tabs. This would require a popup or secondary menu option to allow the user to choose between switching the current view and adding to a new tab. Once that's implemented it's as simple as updating the `Store` with the server selected from the tabview.

**Comment by epsimatic** [comment]
Mac OS have window auto-tiling feature where it creates a new tab in your app for something that's expected to be a new window. Ex. [here](https://support.apple.com/en-gb/guide/mac-help/mchlp2469/mac#mchl7cdb2590): 

> Merge an app's windows into one tabbed window: In the app, choose Window \> Merge All Windows. If an app has more than one type of window (such as Mail with the viewer window and the new message window), only the active type is merged.
> To make a tab a separate window again, select the tab, then choose Window \> Move Tab to New Window, or just drag the tab out of the window. See [Use tabs in windows](https://support.apple.com/en-gb/guide/mac-help/use-tabs-in-windows-mchla4695cce/12.0/mac/12.0).

Also [here](https://support.apple.com/en-gb/guide/mac-help/mchla4695cce/12.0/mac/12.0#mchl4abc0523): 

> Specify when to open documents in tabs
>  1. On your Mac, choose Apple menu  \> System Preferences, then click General .
>  2. Click the "Prefer tabs when opening documents" pop-up menu, then choose an option --- never, in full screen or always.

I believe you can opt in your app to always have your app windows tabbed, effectively making the OS to do the job for you

**Comment by TheNightmanCodeth** [comment]
@epsimatic yeah looks like it already works! I'd like to have the server name show in the tab name so I'll get to work on that for the complete implementation
